core/src/main/java/io/smallrye/openapi/runtime/io/securityrequirement/SecurityRequirementConstant.java

```diff
@@ -17,6 +19,8 @@
 
     static final DotName DOTNAME_SECURITY_REQUIREMENT = DotName.createSimple(SecurityRequirement.class.getName());
     static final DotName DOTNAME_SECURITY_REQUIREMENTS = DotName.createSimple(SecurityRequirements.class.getName());
+    static final DotName DOTNAME_SECURITY_REQUIREMENTS_SET = DotName.createSimple(SecurityRequirementsSet.class.getName());
+    static final DotName DOTNAME_SECURITY_REQUIREMENTS_SETS = DotName.createSimple(SecurityRequirementsSets.class.getName());
```

Comment:
If the string constant values are used here instead of `Class#getName`, I'm curious if this will pass the 3.0 or even 2.0 MP OpenAPI TCK. If it does, we can merge earlier since Jandex buffers us a bit from API changes.